modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -137,15 +157,17 @@ nginx   LoadBalancer   10.43.183.104   192.168.1.241   81:32434/TCP   2m
 
 .Verification
 
-* The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service configuration. The result of the command is a list of those server IP addresses. Verify that the load balancer sends requests to all the running applications with the following command:
+The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service configuration. The result of the command is a list of those server IP addresses.
+
+* Verify that the load balancer sends requests to all the running applications with the following command:
```

Comment:
Nit - should this wording be "by running the following command" or something similar?
```suggestion
* Verify that the load balancer sends requests to all the running applications by running the following command:
```